AutomatedTesting/Gem/PythonTests/assetpipeline/scene_tests/scene_tests.py

```diff
@@ -0,0 +1,185 @@
+    ap_setup_fixture["tests_dir"] = os.path.dirname(os.path.realpath(__file__))
+
+
+@pytest.mark.usefixtures("asset_processor")
+@pytest.mark.usefixtures("ap_setup_fixture")
+@pytest.mark.usefixtures("local_resources")
+@pytest.mark.parametrize("project", targetProjects)
+@pytest.mark.assetpipeline
+@pytest.mark.SUITE_sandbox
+class TestScene_AllPlatforms(object):
+
+    @pytest.mark.BAT
+    @pytest.mark.parametrize("blackbox_param", blackbox_scene_tests, ids=blackbox_test_ids)
+    def test_SceneBlackboxTest_SourceFiles_Processed_ResultInExpectedProducts(self, workspace: AbstractWorkspaceManager,
+                                                                              ap_setup_fixture: dict,
+                                                                              asset_processor: object, project: any,
+                                                                              blackbox_param: ParameterSet) -> None:
+        """
+        Please see run_fbx_test(...) for details
+        Test Steps:
+        1. Determine if blackbox is set to none
+        2. Run FBX Test
+        3. Determine if override assets is set to none
+        4. If not, re-run FBX test and validate the information in override assets
+        """
+
+        self.run_scene_test(workspace, ap_setup_fixture, asset_processor, project, blackbox_param)
+
+        if blackbox_param.override_assets is not None:
+            # Run the test again and validate the information in the override assets
+            self.run_scene_test(workspace, ap_setup_fixture,
+                                asset_processor, project, blackbox_param, True)
+
+
+    @staticmethod
+    def run_ap_debug_skip_atom_output(asset_processor: object) -> None:
+        """Helper: Run Asset Processor with debug output enabled and Atom output disabled."""
+        result, output = asset_processor.batch_process(capture_output=True, extra_params=["--debugOutput",
+                                                                                          "--regset=\"/O3DE/SceneAPI/AssetImporter/SkipAtomOutput=true\""])
+        # If the test fails, it's helpful to have the output from asset processor in the logs, to track the failure down.
+        logger.info(f"Asset Processor Output: {pformat(output)}")
+        assert result, "Asset Processor Failed"
+
+    @staticmethod
+    def check_missing_assets(expected_product_list: list, cache_folder: str) -> None:
+        """Helper: Compare expected products with products in cache, report on missing products."""
+        missing_assets, _ = utils.compare_assets_with_cache(expected_product_list,
+                                                            cache_folder)
+
+        # If the test is going to fail, print information to help track down the cause of failure.
+        if missing_assets:
+            in_cache = os.listdir(cache_folder)
+            logger.info(f"The following assets were missing from cache: {pformat(missing_assets)}")
```

Comment:
Nit: Maybe use logger.debug if these messages are for debugging failures?

Reply:
Added to list of phase 2 fixes.